File: 2A_2/cpp/suma_parzyste.cpp
/*
 * suma_parzyste.cpp
 * <0, 100>
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int suma, liczba;
	// "liczba" to zmienna sterująca
	int start, koniec;
	start = koniec = 0;
	
	cout << "Podaj liczbę początkową: ";
	cin >> start;

	cout << "Podaj liczbę końcową: ";
	cin >> koniec;

	suma = 0;
	for (liczba = start; liczba < koniec + 1; liczba++) {
		if (liczba % 2 == 0)
			suma = suma + liczba;
	}
	cout << suma;
	return 0;
}

File: 2A_1/python/suma_parzyste.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  suma_parzyste.py
#  
#  Program sumuje liczby parzyste z wygenerowanego zakresu


def main(args):
    suma = 0
    start = int(input("Podaj początek zakresu: "))
    koniec = int(input("Podaj koniec zakresu: "))
    
    for liczba in range(start, koniec + 1):
        print(liczba)
        if liczba % 2 == 0:
            suma = suma + liczba

    print("Suma =", suma)
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
